Assets/Scripts/Game/Guilds/Guild.cs

```diff
@@ -81,7 +81,7 @@ public virtual TextFile.Token[] UpdateRank(PlayerEntity playerEntity)
                         tokens = TokensPromotion(newRank);
                     else if (newRank < 0)       // Expulsion
                         tokens = TokensExpulsion();
-                    else if (newRank < rank)    // Demotion
+                    else                        // Demotion
```

Comment:
I would prefer this is left as an explicit condition please.

Reply:
OK, I'll put it back to how it was.